forge/db/migrations/20250225-01-EE-add-externalUrl-to-flow-templates.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Add inferred Payload schema to topics
+ */
+const { DataTypes } = require('sequelize')
```

Comment:
```suggestion
/* eslint-disable no-unused-vars */

/**
 * Add inferred Payload schema to topics
 */
const { DataTypes, QueryInterface } = require('sequelize')
```

Ensure types are loaded for `context`